Docs/Help/Set-TeamViewerAPIUri.md

```diff
@@ -0,0 +1,84 @@
+---
+external help file: TeamViewerPS-help.xml
+Module Name: TeamViewerPS
+online version: https://github.com/teamviewer/TeamViewerPS/blob/main/Docs/Help/Set-TeamViewerAPIUri.md
+schema: 2.0.0
+---
+
+# Set-TeamViewerAPIUri
+
+## SYNOPSIS
+
+Change TeamViewer WebAPI Uri.
+
+## SYNTAX
+
+```powershell
+Set-TeamViewerAPIUri [-NewUri <String>] 
+```
+
+## DESCRIPTION
+
+Changes the WebAPI Uri for internal testing.
```

Comment:
Change uri of TeamViewer web API for TeamViewerPS for TeamViewer internal testing purposes.